source/write-to-mongodb.txt

```diff
@@ -27,3 +27,8 @@ Write to MongoDB
        content: |
 
          .. include:: /scala/write-to-mongodb.txt
+
+.. important::
+
+   If your write operation includes a field with a ``null`` value,
+   the ``null`` value is written to MongoDB. 
```

Comment:
Suggestion: I think I understand this, but added some detail in my suggestion below to check my assumption and avoid passive voice.

```suggestion
   the connector writes the field name and ``null`` value to MongoDB. 
```